package.json

```diff
@@ -16,6 +16,7 @@
     "application-config-path": "^0.1.0",
     "chromecasts": "^1.8.0",
     "create-torrent": "^3.22.1",
+    "dlnacasts": "^0.0.1",
```

Comment:
I didn't find any complete solution with a good API, so i made it based on chromecasts